daemon-systemd.in

```diff
@@ -10,8 +10,8 @@ PIDFile=@lockfile@
 PermissionsStartOnly=true
 ExecStartPre=-/usr/bin/mkdir /var/run/naemon
 ExecStartPre=/usr/bin/chown -R naemon:naemon /var/run/naemon/
-ExecStartPre=su - naemon -c "@bindir@/naemon -vp @pkgconfdir@/naemon.cfg"
-ExecStart=@bindir@/naemon -d @pkgconfdir@/naemon.cfg
+ExecStartPre=su - naemon -s /bin/sh -c "@bindir@/naemon -vp @pkgconfdir@/naemon.cfg"
```

Comment:
I would like it if we used the long options `--login`, `--shell`, `--command`, `--verify-config`, `--precache-objects` for clearness purposes.